Dockerfile

```diff
@@ -27,32 +27,29 @@ RUN pnpm install
 
 COPY . /code/
 
-# -------------------------- Nginx - Builder --------------------------------
-FROM builder AS nginx-build
+FROM builder AS web-app-serve-build
 
 # Dynamic configs. Can be changed with containers. (Placeholder values)
-ENV APP_TITLE=APP_TITLE_PLACEHOLDER
-ENV APP_ENVIRONMENT=APP_ENVIRONMENT_PLACEHOLDER
-ENV APP_MAPBOX_ACCESS_TOKEN=APP_MAPBOX_ACCESS_TOKEN_PLACEHOLDER
-ENV APP_GOOGLE_ANALYTICS_ID=APP_GOOGLE_ANALYTICS_ID_PLACEHOLDER
-ENV APP_GRAPHQL_API_ENDPOINT=https://APP-GRAPHQL-API-ENDPOINT-PLACEHOLDER.COM/
-ENV APP_HCAPTCHA_SITEKEY=APP_HCAPTCHA_SITEKEY_PLACEHOLDER
+ENV APP_TITLE=Alerthub
+ENV APP_ENVIRONMENT=production
+ENV APP_MAPBOX_ACCESS_TOKEN=WEB_APP_SERVE_PLACEHOLDER__APP_MAPBOX_ACCESS_TOKEN
```

Comment:
Don't do this, use a dummy data or leave it empty.
We only need to do this to variables that are used outside of the application itself.